docs/content/en/docs/installation/compatibility.md

```diff
@@ -5,14 +5,29 @@ description: >
   Fission Compatibility with environments, Keda and Keda Connectors
 ---
 
-| Fission version | Keda version | Keda Connectors image tag     |
-| --------------- | ------------ | ------------------------------|
-| 1.11.2          | 1.5          | v0.1                          |
-| 1.12.0          | 2.0          | v0.5                          |
-| 1.13.1          | 2.0          | v0.6 (except GCP pub-sub v0.1)|
-| 1.14.0          | 2.0          | v0.7 (except GCP pub-sub v0.2)|
+| Fission version | Keda version | Keda Connectors image tag      |
+| --------------- | ------------ | ------------------------------ |
+| 1.11.2          | 1.5          | v0.1                           |
+| 1.12.0          | 2.0          | v0.5                           |
+| 1.13.1          | 2.0          | v0.6 (except GCP pub-sub v0.1) |
+| 1.14.0          | 2.0          | v0.7 (except GCP pub-sub v0.2) |
```

Comment:
I think we should call it 2.0 + so people don't stick to just 2.0 version itself!